features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx

```diff
@@ -64,6 +66,10 @@ const styles = (theme) => ({
     buttonIcon: {
         marginRight: 10,
     },
+    buttonWarningColor: {
+        marginRight: 10,
+        color: '#ffcc00',
```

Comment:
Shall we use a color from theme ?

Example: [Codesandbox](https://codesandbox.io/s/material-demo-forked-4sgvo?file=/demo.js)

Reply:
Fixed